TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/ItemsListFragment.kt

```diff
@@ -50,42 +49,73 @@ class ItemsListFragment : Fragment(), AdapterView.OnItemClickListener {
         }
 
         view.findViewById<FloatingActionButton>(R.id.add_fab).setOnClickListener {
-            addItem()
+            DataProvider.createNote()
+
+            // Set selected item to newly created note (first element in list)
+            setSelectedItem(0)
```

Comment:
maybe just have `startNoteFragment(0)` here instead. `setSelectedItem(position)` is called in `startNoteFragment()`